internal/server/server.go

```diff
@@ -51,6 +51,9 @@ type Options struct {
 	// Redis contains configuration options to the cache server.
 	Redis redis.Options
 
+	GoogleClientID     string
+	GoogleClientSecret string
```

Comment:
minor: I've been trying to keep any new config settings off the top level config. If we are planning on adding more managed providers maybe we can put this under a `ManagedProviders` field, and have:

```go
type ManagedProviders struct {
	GoogleClientID     string
	GoogleClientSecret string
}
```

Reply:
We dont have plans to add more for the time being, let's move it under a struct if we do